src/writer.rs

```diff
@@ -57,6 +57,9 @@ pub struct WriterOptions<'a> {
     /// The header file will only be compressed if the caller specifically asked for a path ending
     /// with "hdr.gz". Otherwise, only the volume will be compressed (if requested).
     force_header_compression: bool,
+
+    /// optional ExtensionSequence
```

Comment:
Can you please check all new doc `///` in this MR and ensure that they start with an upper case character?

In fact, I would prefer if you do it for all your new comments `//` but this is less important.